### PR TITLE
Update dotnet

### PR DESCRIPTION
**Pull request by troydai**

### CODE REVIEW

File: build-cli.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
```
Comment (Mpdreamz) [inline build-cli.py]: Nothing against python per se but this introduces a python dep for something that could be replaced in the build.cmd/sh at first glance.

Comment (troydai) [inline build-cli.py]: Thanks for point out. I’ll eventually turn this script in to ps1 and sh.

However, I find it’s nice to have one script across the platform. If we can agree on something that will be great.

File: src/OmniSharp.CliHost/Program.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using OmniSharp.Host;
+
+namespace OmniSharp.CliHost
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            Console.WriteLine($"OmniSharp-CLI: {string.Join(" ", args)}");
```
Comment (davidfowl) [inline src/OmniSharp.CliHost/Program.cs]: Remove

**Comment by troydai** [comment]
I'll begin working on this branch tomorrow.

**Comment by troydai** [comment]
Move to https://github.com/OmniSharp/omnisharp-roslyn/pull/398

**Pull request closed without merging**